Aqovia.Utilities.SagaMachine/StatePersistance/IKeyValueStore.cs

```diff
@@ -8,6 +8,9 @@ public interface IKeyValueStore
         bool TrySetValue<T>(string key, T value, string oldHash);
         bool Remove(string key, string oldHash);
         void Remove(string key);
+        bool TakeLockWithShortTimeSpan(string key, out string lockToken);
```

Comment:
Just the TimeSpan in the name leaves me wondering whether it's about a timeout to acquire the lock (like in .NET synchronization primitives) or a token expiry date. Since Redis calls it 'expiry', perhaps worth saying the same.

Also, `ShortTimeSpan` begs the question what is considered short. How about just saying 'default' instead.
So, something like:
`TakeLockWithDefaultExpiryTime`